apps/embers/src/pages/CreateDashboard/CreateDashboard.tsx

```diff
@@ -0,0 +1,86 @@
+import { Text } from "@/lib/components/Text";
+import { ThemeSwitch } from "@/lib/components/ThemeSwitch";
+
+import ChevronIcon from "../../public/icons/icon_components/chevron-icon";
```

Comment:
@ points to root directory
```suggestion
import ChevronIcon from "@/public/icons/icon_components/chevron-icon";
```

Reply:
Thanks